modules/migration-cleaning-up-failed-migration.adoc

```diff
@@ -0,0 +1,132 @@
+
+
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc
+
+
+:_module-type: PROCEDURE
+
+[id="cleanup-of-failed-migration_{context}"]
+= Cleaning up a failed migration
+
+.Prerequisites
+* You must be logged in to the {product-title} cluster as a user with the `cluster-admin` role.
+* You must have the OpenShift CLI installed.
+
+
+.Procedure
+
+
+. Deleting resources
++
+If a migration fails during Stage or Copy, some resources are retained to allow debugging and need to be cleaned up.
++
+The following resources must be deleted:
+
+* rsync/stunnel pods, both in the source and target namespaces
```

Comment:
Pod names should be correctly capitalized and marked up. No slashes.

Reply:
Did you check the current docs? In enterprise-4.8, I recently merged a [PR](https://github.com/openshift/openshift-docs/pull/34418) for leftover migration resources.

Reply:
rsync is used for DVM, so you should mention that. Not used for all migrations